doc/source/whatsnew/v1.0.0.rst

```diff
@@ -115,10 +115,10 @@ Datetimelike
 - Bug in :meth:`Series.__setitem__` incorrectly casting ``np.timedelta64("NaT")`` to ``np.datetime64("NaT")`` when inserting into a :class:`Series` with datetime64 dtype (:issue:`27311`)
 - Bug in :meth:`Series.dt` property lookups when the underlying data is read-only (:issue:`27529`)
 - Bug in ``HDFStore.__getitem__`` incorrectly reading tz attribute created in Python 2 (:issue:`26443`)
+- Bug in :func:`to_datetime` where passing arrays of malformed ``str`` while attempting to coerce errors could incorrectly lead to raising errors instead (:issue:`28299`)
```

Comment:
'while attempting to coerce errors' --> 'with `errors="coerce"`'

'could incorrectly lead to raising errors instead' --> 'could incorrectly raise [???]`